site2/docs/reference-pulsar-admin.md

```diff
@@ -1170,7 +1170,7 @@ $ pulsar-admin namespaces set-message-ttl tenant/namespace options
 Options
 |Flag|Description|Default|
 |----|---|---|
-|`-ttl`, `--messageTTL`|Message TTL in seconds. When the value is set to `0`, TTL is disabled. TTL is disabled by default. |0|
+|`-ttl`, `--messageTTL`|Message TTL in seconds|3600s|
```

Comment:
It seems that there are many places to confirm according to the code. The default value here should be 0 (disabled),because the default values of topic and namespace-level are both null (not set)

we can see the broker-level default value in broker.conf (ttlDurationDefaultInSeconds)
namespace-level default value in `pulsar/pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/Policies.java`
topic-level default in `pulsar/pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TopicPolicies.java`

Reply:
The default values and description have been fixed in https://github.com/apache/pulsar/pull/9395
So I sincerely suggest that we rebase the master first and then make edits.